turkle/models.py

```diff
@@ -8,7 +8,7 @@
 import sys
 
 from bs4 import BeautifulSoup
-from django.contrib.auth.models import User
+from django.contrib.auth import get_user_model
```

Comment:
Take a look at this post: https://stackoverflow.com/questions/24629705/django-using-get-user-model-vs-settings-auth-user-model

Reading that makes me think that we want to use settings.AUTH_USER_MODEL rather than get_user_model() when defining foreign keys.